src/OneDriveSdk/Exceptions/Error.cs

```diff
@@ -39,6 +39,9 @@ public class Error
         [DataMember(Name = "message", IsRequired = false, EmitDefaultValue = false)]
         public string Message { get; set; }
 
+        [DataMember(Name = "throwSite", IsRequired = false, EmitDefaultValue = false)]
+        public string ThrowSite { get; set; }
```

Comment:
We should add a ToString function to make sure this debugging throwsite is included in most views of the exception to help ferry it into issue reports